db/schema.rb

```diff
@@ -152,13 +152,13 @@
     t.index ["user_id"], name: "index_subscriptions_on_user_id", unique: true
   end
 
-  create_table "users", id: :serial, force: :cascade do |t|
-    t.string "password_digest", limit: 255
-    t.datetime "created_at", precision: nil, null: false
-    t.datetime "updated_at", precision: nil, null: false
-    t.string "api_key", limit: 255, null: false
-    t.string "username", null: false
+  create_table "users", force: :cascade do |t|
+    t.string "password_digest"
```

Comment:
Looks like this changed as well. Probably need a separate migration to remove the limit here.